test/models/category_test.rb

```diff
@@ -1,7 +1,49 @@
 require "test_helper"
 
 class CategoryTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    DatabaseCleaner.start
+    site = create(:site)
+    @categories = create_list(:category, 5, site: site)
```

Comment:
When the some category names are the same as other one, the test cases of category's order are silently broken :<